backend/migrations/versions/5a798acc6282_create_version_mf.py

```diff
@@ -0,0 +1,101 @@
+    # ### commands auto generated by Alembic - please adjust! ###
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+
+    op.create_table(
+        'metadata_form_version',
+        sa.Column('metadataFormUri', sa.String(), nullable=False),
+        sa.Column('version', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ['metadataFormUri'], ['metadata_form.uri'], name='f_key_version_metadata', ondelete='CASCADE'
+        ),
+        sa.PrimaryKeyConstraint('metadataFormUri', 'version'),
+    )
+    for mf in session.query(MetadataForm).all():
+        version = MetadataFormVersion(metadataFormUri=mf.uri, version=1)
+        session.add(version)
+        session.commit()
+
+    op.add_column('attached_metadata_form', sa.Column('version', sa.Integer()))
+    for amf in session.query(AttachedMetadataForm).all():
+        amf.version = 1
+        session.commit()
+    op.alter_column('attached_metadata_form', 'version', nullable=False)
+    op.create_foreign_key(
+        'fk_attached_mf_version_uri',
+        'attached_metadata_form',
+        'metadata_form_version',
+        ['metadataFormUri', 'version'],
+        ['metadataFormUri', 'version'],
+        ondelete='CASCADE',
+    )
+
+    op.add_column('metadata_form_enforcement_rule', sa.Column('version', sa.Integer(), nullable=False))
+    op.create_foreign_key(
+        'f_key_enforcement_version_metadata',
+        'metadata_form_enforcement_rule',
+        'metadata_form_version',
+        ['metadataFormUri', 'version'],
+        ['metadataFormUri', 'version'],
+        ondelete='CASCADE',
+    )
+
+    op.add_column('metadata_form_field', sa.Column('version', sa.Integer()))
+    for field in session.query(MetadataFormField).all():
+        field.version = 1
+        session.commit()
+    op.alter_column('metadata_form_field', 'version', nullable=False)
+    op.create_foreign_key(
+        'fk_version',
+        'metadata_form_field',
+        'metadata_form_version',
+        ['metadataFormUri', 'version'],
+        ['metadataFormUri', 'version'],
+        ondelete='CASCADE',
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint('fk_version', 'metadata_form_field', type_='foreignkey')
+    op.drop_column('metadata_form_field', 'version')
+    op.drop_constraint('f_key_enforcement_version_metadata', 'metadata_form_enforcement_rule', type_='foreignkey')
+    op.drop_column('metadata_form_enforcement_rule', 'version')
+    op.drop_constraint('fk_attached_mf_version_uri', 'attached_metadata_form', type_='foreignkey')
+    op.drop_column('attached_metadata_form', 'version')
+    op.create_table(
+        'maintenance',
+        sa.Column('status', sa.VARCHAR(), autoincrement=False, nullable=False),
+        sa.Column('mode', sa.VARCHAR(), autoincrement=False, nullable=True),
+        sa.PrimaryKeyConstraint('status', name='maintenance_pkey'),
+    )
```

Comment:
I think this should be removed

Reply:
Oh, yes. Removed it from update, forgot about downgrade